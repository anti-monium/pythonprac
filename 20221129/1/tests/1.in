class C(metaclass=dump):
	def __init__(self, val):
		self.val = val + 13

	def pr_pls(self):
		print('Your welcome!', self.val)

	def add_s(self, s):
		setattr(self, 's', s)

c = C(10)
c.pr_pls()
c.add_s(':)')
