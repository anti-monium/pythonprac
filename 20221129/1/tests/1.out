__init__: (10,), {}
pr_pls: (), {}
Your welcome! 23
add_s: (':)',), {}
